lib/OpenQA/BuildResults.pm

```diff
@@ -106,8 +107,9 @@ sub compute_build_results {
         }
         $jr{reviewed_all_passed} = $jr{passed} == $count;
         $jr{total}               = $count;
-        $jr{reviewed}            = $jr{failed} > 0 && $jr{labeled} == $jr{failed};
-        $builds{$b}              = \%jr;
+        my $failed = $jr{failed} + $jr{softfailed};
```

Comment:
well, yes. A softfailed was supposed to be a passed with small glitches - but in the meantime we treat it as a failed with workarounds in place.